tools/windows-node-installer/README.md

```diff
@@ -24,86 +24,41 @@ make build-tools
 ```
 
 ## How to use it
-### Creating a Windows instance:
-
-```bash
-wni --help
-Creates and destroys Windows instance with an existing OpenShift cluster.
-
-Usage:
-  wni [command]
 
-Available Commands:
-  aws         Takes aws specific resource names from user
-  help        Help about any command
+The `wni` requires the kubeconfig of the OpenShift cluster, a provider specific credentials file to create and 
+destroy a Windows instance on the selected provider. To create an instance, `wni` also 
+requires extra information such as image id and instance type. Some optional flags include directory path to 
+windows-node-installer.json file and log level display. For more information please 
+visit `--help` for any commands or sub-commands.
 
-Flags:
-      --dir string          directory to save or read window-node-installer.json file from. (default ".")
-  -h, --help                help for wni
-      --kubeconfig string   file path to the kubeconfig of the existing OpenShift cluster (required).
-      --log-level string    log level (e.g. 'info') (default "info")
-
-Use "wni [command] --help" for more information about a command.
-```
-
-For example for aws:
+### Creating a Windows instance:
```

Comment:
using AWS - I think you need to call this out as Azure implementation might have section similar to this.